src/clj/harja/palvelin/raportointi/raportit/ymparisto.clj

```diff
@@ -88,6 +88,7 @@
 
     [:raportti {:nimi raportin-nimi
                 :orientaatio :landscape}
+     [:teksti "Raportti ei vielä kykene laskemaan Suolan toteumaprosentteja."]
```

Comment:
Lisäsin viestin, koska tuosta tulee jatkuvasti palautetta. Käyttäjät varmaan luulevat, että sen PITÄISI toimia, mutta bugaa. Näinhän ei ole.

En kyl tiedä, milloin ominaisuus lisätään. Monestihan tuosta on taidettu asiakkaan kanssa puhua, ja aina puskettu prioriteettia alaspäin. Voisi varmaan muokata viestiä, että yksinkertaisesti "ei laske".